unison-cli/src/Unison/Codebase/Editor/HandleInput/FindAndReplace.hs

```diff
@@ -91,6 +93,47 @@ handleStructuredFindI rule = do
   Cli.setNumberedArgs $ map SA.HashQualified results
   Cli.respond (ListStructuredFind results)
 
+handleTextFindI :: Bool -> [String] -> Cli ()
+handleTextFindI allowLib tokens = do
+  Cli.Env {codebase} <- ask
+  currentBranch <- Cli.getCurrentBranch0
+  hqLength <- Cli.runTransaction Codebase.hashLength
+  let names = Branch.toNames currentBranch
```

Comment:
There's `Cli.currentNames` or `Cli.projectBranchNames` for this btw, best if we use those because we may choose to add caching to those calls in the future.